includes/class-solrpower-wp-query.php

```diff
@@ -740,7 +740,7 @@ private function parse_tax_query( $tax_query ) {
 					$multi_query   = array();
 					$multi_query[] = '(' . $field . ':(' . implode( 'OR', $terms ) . '))';
 					if ( $tax_value['include_children'] && is_taxonomy_hierarchical( $tax_value['taxonomy'] ) ) {
-						$multi_query[] = '(parent_' . $field . ':' . implode( 'OR', $terms ) . ')';
+						$multi_query[] = '(parent_' . $field . ':' . implode( 'OR', $terms ) . '))';
```

Comment:
you're missing the sad emoji in the middle 
```suggestion
						$multi_query[] = '(parent_' . $field . '(:' . implode( 'OR', $terms ) . '))';
```

from https://wordpress.org/support/topic/issue-building-query-when-you-are-parsing-the-tax_query/